interfaces/apparmor/template.go

```diff
@@ -233,13 +233,15 @@ var defaultTemplate = []byte(`
 
   # Writable home area for this version.
   owner @{HOME}/snap/@{SNAP_NAME}/@{SNAP_REVISION}/** wl,
+  owner @{HOME}/snap/@{SNAP_NAME}/shared/** wl,
```

Comment:
> shared is perhaps not the best word. It also means "make it public" (share with whom?), and it may imply concurrent use which will never be the case here (only the active snap will touch it).

Great point, I agree.

> As a preliminary strawman, I'd suggest "common".

I'm good with "common." How about the variable name format: `SNAP_COMMON_DATA` and `SNAP_USER_COMMON_DATA`? I'm concerned the latter gives the impression that the data is common among users, which of course isn't the case. Can you think of something more clear?